### DISCUSSION TITLE
Does it work on America servers?

### DISCUSSION THREAD

**Discussion by vsinghh86** [adipiciu/Travian-scripts#73] [Q&A] [answered]
I tried running it on ts4.x1.america.travian.com. Only the task queue script works, resource bar script does not work. It installs but then nothing. It worked on asia servers though.

**Comment by adipiciu** [comment]
It doesn't work. Maybe the last travian update broke something. I will investigate and update later.

**Comment by adipiciu** [comment, marked as answer]
@vsinghh86 I fixed the issue. Check the latest script version 2.24.12

**Reply by vsinghh86** [reply]
Yes it works on America now, Thanks a lot.
